src/pages/form-validation/password/index.tsx

```diff
@@ -0,0 +1,435 @@
+                                fullWidth
+                                variant={'filled'}
+                                InputProps={{
+                                    endAdornment: (
+                                        <InputAdornment position={'end'}>
+                                            <IconButton
+                                                style={{ height: 36, width: 36 }}
+                                                onClick={(): void => setShowNewPassword(!showNewPassword)}
+                                            >
+                                                {showNewPassword && <Visibility />}
+                                                {!showNewPassword && <VisibilityOff />}
+                                            </IconButton>
+                                        </InputAdornment>
+                                    ),
+                                }}
+                            />
+
+                            <List disablePadding component={'ul'} style={{ marginTop: 8 }}>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.minLengthRequired)}
+                                    <Typography variant={'caption'}>At least 8 characters in length</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1NumberRequired)}
+                                    <Typography variant={'caption'}>At least 1 digit</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1UpperCharRequired)}
+                                    <Typography variant={'caption'}>At least 1 uppercase letter</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1LowerCharRequired)}
+                                    <Typography variant={'caption'}>At least 1 lowercase letter</Typography>
+                                </ListItem>
+                                <ListItem disableGutters className={classes.passwordCriteria}>
+                                    {getPasswordCriteriaIcon(passwordErrors.atLeast1SplCharRequired)}
+                                    <Typography variant={'caption'}>
+                                        At least 1 special character: (valid: ! @ # $ ^ &)
+                                    </Typography>
+                                </ListItem>
+                            </List>
```

Comment:
You should steal the subcomponents from the login workflow and use those here instead of reinventing the wheel (you will just need to tweak the font size - we talked about this discrepancy between the workflows and the DSM a fews weeks back). 

The color should also change to  gray when the criteria is met (if you take the existing implementation, this will already be correct).